test/general/header_inclusion_order_memory_0.pass.cpp

```diff
@@ -20,9 +20,22 @@
 
 #include "support/utils.h"
 
+#if ONEDPL_HAS_RANGE_ALGORITHMS > 202505L
+#    include <ranges>
+#endif
+
 int
 main()
 {
-
+#if ONEDPL_HAS_RANGE_ALGORITHMS > 202505L
+    constexpr int n = 10;
+    std::allocator<int> alloc;
+    int* raw_ptr = alloc.allocate(n);
+    auto deleter = [&](int* p){ alloc.deallocate(p, n); };
+    std::unique_ptr<int, decltype(deleter)> ptr(raw_ptr, deleter);
+    std::ranges::subrange subrange(ptr.get(), ptr.get() + n);
+    oneapi::dpl::ranges::uninitialized_fill(oneapi::dpl::execution::seq, subrange, 0);
```

Comment:
This test is bad in my view. You fill a raw memory with zeros but the compiler itself could fill the raw memory with zeros for easier debuging. Furthermore, it does not matter because you don't check anything calling `uninitialized_fill`


```suggestion
    oneapi::dpl::ranges::uninitialized_fill(oneapi::dpl::execution::seq, subrange, 42);
    EXPECT_TRUE(std::ranges::count(subrange, 42) == n, "some text");
```

But this is just a minimal suggestion. We are supposed to have a more exhaustive discussion about `<memory>` tests on Thursday, where I hope we will develop a common approach about what we want from our tests for `uninitialized_` family and `destroy`

Reply:
I added this case to solely check if the API is available (hence `header_inclusion_order*` name of the test).
The results are checked in dedicated `std_ranges_uninitialized_fill` test.